spec/truefi2/scripts/BorrowingMutex.sh

```diff
@@ -1,2 +1,3 @@
 certoraRun contracts/truefi2/BorrowingMutex.sol \
-  --verify BorrowingMutex:spec/truefi2/BorrowingMutex.spec
+  --verify BorrowingMutex:spec/truefi2/BorrowingMutex.spec \
+  --rule_sanity
```

Comment:
This creates an additional run for each rule by replacing each `assert x` with `assert false` and makes sure all rules are *violated*. This helps avoid vacuous rules like:
```
rule vacuous {
    uint256 x;
    require x != x;

    assert x > 10;
}
```
https://certora.atlassian.net/wiki/spaces/CPD/pages/41124372/Common+Pitfalls